cylc/flow/scripts/install.py

```diff
@@ -48,16 +49,34 @@ def main(parser, options, reg):
     for entry_point in pkg_resources.iter_entry_points(
         'cylc.pre_configure'
     ):
-        entry_point.resolve()(Path(flow_file).parent)
+        try:
+            entry_point.resolve()(Path(flow_file).parent)
+        except Exception as exc:
+            # NOTE: except Exception (purposefully vague)
+            # this is to separate plugin from core Cylc errors
+            raise PluginError(
+                'cylc.post_configure',
```

Comment:
@oliver-sanders in `fileparse.py`, the entry point iterated and the first argument to `PluginError` both matched (`cylc.pre_configure`).

But here we iterate the plug-ins `cylc.pre_configure` and raise an error `PluginError` with `cylc.post_configure`. Is that correct?

Reply:
Dammit, no, was rushing.